### ISSUE TITLE
Win 10 Flat Color OSD Skin

### ISSUE THREAD

**Issue by mgonzales71** [irzyxa/Volume2#46]
Hi,

Love your app just found it last week.

If interested - just tried my hand at creating a new OSD Skin for Volume2

Do you have any skin dev documents?

Thanks,

Mario

here is a link to the repo

https://github.com/mgonzales71/Volume2.OSD.Skin_Win10FlatColor

I could also clone your repo and do a PR if you prefer... just let me know.

A few preview shots
![volume2_2018-01-16_18-26-50](https://user-images.githubusercontent.com/6510026/35022590-840758c4-faeb-11e7-8895-e77084e892c4.png)
![volume2_2018-01-16_18-24-45](https://user-images.githubusercontent.com/6510026/35022591-84457618-faeb-11e7-9bed-c747b7692386.png)
![volume2_2018-01-16_18-25-27](https://user-images.githubusercontent.com/6510026/35022592-84610ec8-faeb-11e7-8761-ebfb5e55aa96.png)
![volume2_2018-01-16_18-06-39](https://user-images.githubusercontent.com/6510026/35022783-94e7b50c-faec-11e7-8875-a2e125fff5f3.png)

**Comment by irzyxa** [comment]
Cool skin. You can read more there: 
https://github.com/irzyxa/Volume2/tree/master/Docs.
If you don't mind I will include the skin to Volume2 repository.

**Comment by mgonzales71** [comment]
You are certainly welcome to include it - Thanks for the compliment and again very cool app.

I already have a couple other ideas for another skin

one of them is to use in some way the radial ui element used on the apple watch 

Cool web app to generate the images

http://hmaidasani.github.io/RadialChartImageGenerator/

Looks like I will have to break out the old google translator unless I am missing a possible EN version of the doc?  No worries if not - I have a global mindset 😃 hehe

oh and I was curious if the only reason to start Volume2 as admin is to write / modify OSD skin preferences, etc?

maybe in a future update you can do a UAC elevated prompt that uses a startup or on demand service/task to avoid the UAC prompts for future restarts?

Have a great day well its close to night for you now!

Mario

**Comment by mgonzales71** [comment]
I ran this thru a translation app really quickly - just skimmed it but looks like the translation is not that bad.

attaching a copy in case you want to include this with the other.

once I get more time I will be able to read the doc more closely and see how good the translator really did.

[Volume2 Creation Skins (Auto Translated to EN).pdf](https://github.com/irzyxa/Volume2/files/1639371/Volume2.Creation.Skins.Auto.Translated.to.EN.pdf)

**Comment by mgonzales71** [comment]
Just updated my skin and the text size/alignment/position for optical and usb drives/devices

so you may want to grab updated files if you have already made copies, etc.

screenshots follow

Enjoy!

![volume2_2018-01-18_20-28-41](https://user-images.githubusercontent.com/6510026/35134968-d6714d0a-fc8e-11e7-971b-eee0a40dd1bf.png)
![volume2_2018-01-18_20-04-32](https://user-images.githubusercontent.com/6510026/35134969-d68f3ee6-fc8e-11e7-876f-3dd04ccee10d.png)
![volume2_2018-01-18_20-28-06](https://user-images.githubusercontent.com/6510026/35134970-d6bd58f8-fc8e-11e7-9ae5-73cdbbea7163.png)

**Comment by mgonzales71** [comment]
Last update till I make a cool new skin..

Created and Dumb Installer for my skin because I was bored lol
just blindly installs in the default directories so not smart at all

anyway I don't know if you ever wanted to say have user installed sounds and skins be easier and maybe also look for them in %APPDATA%\Volume2\... to persist installs, etc but something to consider in a future release

https://github.com/mgonzales71/Volume2OSDSkin-Windows10Style-FlatColor/releases/tag/1.0

**Comment by mgonzales71** [comment]
![volume2 skin icon](https://user-images.githubusercontent.com/6510026/35190947-2ed8e79c-fe23-11e7-9dd4-d51e03d85703.png)

a quick icon I made derived from what you have in app that I used as skin installer icon

yours freely if you want

**Comment by DrTexxOfficial** [comment]
this skin is absolutely gorgeous! <3

**Comment by DrTexxOfficial** [comment]
@mgonzales71 would it be possible to release a dark version of this theme? I'm absolutely in love with it!

It would look incredible for all the users running Win 10 in dark mode like myself!
![dark mode](https://user-images.githubusercontent.com/23412100/35251535-b7cda08e-002f-11e8-85a1-4e5c19583250.PNG)

**Comment by DrTexxOfficial** [comment]
I quickly tried my hand at it, let me know your thoughts :) 👍
[Windows 10 Flat Color - Dark by DrTexxOfficial.zip](https://github.com/irzyxa/Volume2/files/1654120/Windows.10.Flat.Color.-.Dark.by.DrTexxOfficial.zip)

**Comment by mgonzales71** [comment]
@DrTexxOfficial I am not sure how I managed to miss your question but saw notice that this was closed and that is how I saw your question.  Thanks for the kudos btw.

I noticed you have some stray pixels in your quick attempt - I will work together a dark mode version for you and today/this weekend.

This one I did would work well in Dark Mode - let me know your thoughts 

https://github.com/mgonzales71/Volume2OSDSkin-CompactCarbon

Go ahead and open an issue on that repo so we don't spam this repo's issues :)

Mario

PS

This is another that would work too

https://github.com/mgonzales71/Volume2OSDSkin-FloatingRing
